config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -36,6 +36,41 @@ presubmits:
           - name: ndots
             value: "1"
 
+  - name: pull-cert-manager-bazel-nocache
+    description: Run cert-manager unit tests with Bazel remote-caching disabled
+    always_run: false
+    optional: true
+    context: pull-cert-manager-bazel-nocache
+    max_concurrency: 8
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.6
+    annotations:
+      testgrid-create-test-group: 'false'
+      description: Runs 'bazel test --jobs=1 //...'
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
```

Comment:
In another PR I want to see if I can configure the Bazel scratch (.cache/bazel) directory as a persistent disk backed volume rather than emptydir.